py_zipkin/logging_helper.py

```diff
@@ -156,7 +160,8 @@ def __exit__(self, _exc_type, _exc_value, _exc_traceback):
                 _exc_type.__name__,
                 _exc_value,
             )
-            raise ZipkinError(error)
+            self.tracer.clear()
```

Comment:
This is taken care in `zipkin_span`, no? Now we don't throw an exception anymore, so we'll always go through the cleanup logic

Reply:
Can you add an integration test to verify that?